dataclients/kubernetes/kubernetestest/api.go

```diff
@@ -281,6 +284,10 @@ func initNamespace(kinds map[string][]interface{}) (ns namespace, err error) {
 		return
 	}
 
+	if err = itemsJSON(&ns.endpointslices, kinds["EndpointSlice"]); err != nil {
```

Comment:
Should we model pagination here?

Reply:
It would be nice to have but it's not the problem we need to solve, see also the comment above

Reply:
https://github.com/zalando/skipper/issues/2574